app/kumactl/cmd/install/install_transparent_proxy_test.go

```diff
@@ -38,6 +39,11 @@ var _ = Describe("kumactl install transparent proxy", func() {
 
 			// when
 			err := rootCmd.Execute()
+
+			if given.skip != nil && !given.skip(stdout, stderr) {
```

Comment:
how do we prevent this from suddenly disabling on `CI`? I think there should be a hard force for `CI=true` so some day unnoticed we skip these tests.